src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -109,6 +130,15 @@ Whenever possible, please avoid using ad hoc wording.
 
 Specific EEG fields MUST be present:
 
+{{ MACROS___make_metadata_table(
+   {
+      "EEGReference": "REQUIRED",
+      "SamplingFrequency": "REQUIRED",
```

Comment:
Here we should add something like: "The sampling frequency of data channels that deviate from the main sampling frequency SHOULD be specified in the `channels.tsv`file"

Reply:
```suggestion
      "SamplingFrequency": ("REQUIRED", "The sampling frequency of data channels that deviate from the main sampling frequency SHOULD be specified in the `channels.tsv` file"),
```